RELEASE_NOTES

```diff
@@ -17,12 +17,12 @@ The Big Ones:
 - Add new automatic perspective correction module (https://www.darktable.org/2016/03/a-new-module-for-automatic-perspective-correction/)
 - Liquify tool for all your fancy pixel moving (https://www.darktable.org/2016/04/liquify-liquify/)
 - A new image module to use a Color Look Up Table (CLUT) to change colors in the image (https://www.darktable.org/2016/05/colour-manipulation-with-the-colour-checker-lut-module/)
-- In Highlight reconstruction module, the mode LCh reconstruction was completedy rewritten, and is no longer useless!
+- In Highlight reconstruction module, the mode LCh reconstruction was completely rewritten, and is no longer useless!
 - A new tool, darktable-chart, that goes along with the CLUT module, to create styles for the latter
 - Exposure fusion in the basecurve module to lower the dynamic range of images (https://www.darktable.org/2016/08/compressing-dynamic-range-with-exposure-fusion/)
 - Add new raw overexposure indication (https://www.darktable.org/2016/10/raw-overexposed/)
 - Darkroom: the preview (in top-left corner) is now no longer pre-demosaiced, thus the preview, color picker, histogram show the proper results when there is highlight clipping.
-- Add undo/redo support for the darkroom
+- Add undo/redo support for the darkroom (Ctrl-z to undo, Ctrl-y to redo). This comes in complement to the history stack: undo/redo can walk through several successive changes within the same module (while the history stack only records one entry in this case). Also, undo is able to recover a history stack lost by mistake (e.g. by pressing "compress history" with the bottom of the stack selected).
```

Comment:
Not a native speaker, and I may well have added a french touch, but looking at a dictionary it seems "successive" is the right word here (for "Coming one after the other in a series"), while subsequent applies to one modification coming after another, not the set of modification. Perhaps "consecutive" is more common (synonym for successive)?